core/templates/dev/head/galleries/Gallery.js

```diff
@@ -264,9 +267,11 @@ oppia.controller('SearchBar', [
       $scope.searchBarPlaceholder = $translate.instant(
         'I18N_GALLERY_SEARCH_PLACEHOLDER');
       $scope.categoryButtonText = $translate.instant(
-        $scope.selectionDetails.categories.summary, $scope.translationData);
+        $scope.selectionDetails.categories.summary, $scope.translationData,
+        'messageformat');
```

Comment:
Can you add some documentation here? It took a while for me to understand what's going on.